app/ide-desktop/lib/dashboard/src/appUtils.tsx

```diff
@@ -43,3 +43,17 @@ export const ALL_PATHS_REGEX = new RegExp(
 // ===========
 
 export const SEARCH_PARAMS_PREFIX = 'cloud-ide_'
+
+/**
+ * Build a Subscription URL for a given plan.
+ */
+export function getUpgradeURL(plan: string): string {
+  return SUBSCRIBE_PATH + '?plan=' + plan
+}
+
+/**
+ * Build a Subscription URL for contacting sales.
+ */
+export function getContactSalesURL(): string {
```

Comment:
They're here because they're related to Paths(and all paths are specified here). Eventually, we'll move the URLs closer to their pages and the function will follow.

Reply:
> all paths are specified here

correction: all paths *used in `App.tsx`* are specified here. the paths are (or at least, should be) used nowhere outside of `App.tsx`

Reply:
> correction: all paths used in App.tsx are specified here. the paths are (or at least, should be) used nowhere outside of App.tsx

Since we don't specify paths outside of the `App.tsx` this correction makes no change

Reply:
`getContactSalesURL` is used only in `getComponentForPlan.tsx` and `UpgradeButton.tsx`, neither of which are `App.tsx` - unless i'm misunderstanding something.

Reply:
We keep this function with the URL just because we don't want to forget it when we refactor the Router-related stuff and put to it's own place under `/pages`

Reply:
But if you have a better option - I'll comply